src/sagemaker_training/mpi.py

```diff
@@ -334,6 +336,27 @@ def _create_command(self):
         for name in self._env_vars:
             command.extend(["-x", name])
 
+        def is_smddp_installed():
+            try:
+                import smdistributed.dataparallel.torch.torch_smddp
+                output = subprocess.run(
+                    ["which", "smddpmprun"],
+                    capture_output=True,
+                    text=True,
+                    check=True,
+                )
+                return output.stdout != ''
+            except (ImportError, subprocess.CalledProcessError):
+                return False
+
+        env = environment.Environment()
```

Comment:
Add a comment here to explain the purpose of this.

Reply:
Updated it.